test/test.js

```diff
@@ -100,13 +100,13 @@ describe('Magister', function() {
 			...options,
 			username: 'xxx',
 			password: 'xxx',
-		})).to.be.rejectedWith(magisterjs.AuthError)
```

Comment:
Is it still possible to distinguish authentication errors from general errors?

Reply:
My library returns a generic error, so it's not possible in the current state of the commit. I can wrap the error in an autherror if that's necessary

Reply:
It's fine for now, but maybe nice in the future. We're lucky the library is still in alpha state so the API isn't fixed. But I think it's nice to add the AuthError before releasing v2 as stable.